src/game/items.c

```diff
@@ -764,7 +764,12 @@ void Item_TakeDamage(ITEM_INFO *item, int16_t damage, bool hit_status)
     }
 }
 
-bool Item_TestFrame(ITEM_INFO *item, int16_t frame)
+bool Item_TestFrameEqual(ITEM_INFO *item, int16_t frame)
 {
     return item->frame_number == g_Anims[item->anim_number].frame_base + frame;
 }
+
+bool Item_TestFrameRange(ITEM_INFO *item, int16_t start, int16_t end)
+{
+    return item->frame_number >= start && item->frame_number <= end;
```

Comment:
I think both functions should take frames relative to the animation base so that they have the same contract.